### PR TITLE
wayland: 1.18.0 -> 1.19.0

### PR DESCRIPTION
**Pull request by tadeokondrak**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/libraries/wayland/default.nix [hunk 1]
```diff
@@ -28,19 +28,14 @@ let
...
 
   src = fetchurl {
     url = "https://wayland.freedesktop.org/releases/${pname}-${version}.tar.xz";
-    sha256 = "0k995rn96xkplrapz5k648j651wc43kq817xk1x8280h16gsfxa6";
+    sha256 = "sha256-uszZAjANNUWBzVrTzEnapJIdVftBaliD4hh1D+8WbRU=";
   };
 
   patches = [
-    # Fix documentation to be reproducible.
-    (fetchpatch {
```
Comment (SuperSandro2000) [inline pkgs/development/libraries/wayland/default.nix:40]: Please remove fetchpatch from inputs.

**Comment by primeos** [comment]
This should target `staging` and we already updated `wayland` there: #107324 (staging-next should be merged soon).

**Comment by tadeokondrak** [comment]
Sorry about that. I searched for it multiple times here but it didn't show up

**Comment by primeos** [comment]
Ok, no problem. The GitHub search really isn't that great...

**Pull request closed without merging**